src/views/static/js/search.js

```diff
@@ -1,61 +1,69 @@
-                    $("#neutral").css('display', 'flex');
-                }
-            } else if (result.result.status === 'whitelisted') {
-                hideEverything();
-                var strLinkWhitelisted = '';
-                $("#verifiedmessage").html('<b>' + encodeURI($("input").val().toLowerCase().replace('http://','').replace('https://','').replace('www.','').split(/[/?#]/)[0]) + '</b> is a whitelisted address. You can trust it.');
-                strLinkWhitelisted = '<a id="details" href="/address/' + encodeURI($("input").val()) + '">Details on this address <i class="chevron right small icon"></i></a>';
-                $("#verifiedmessage").html($("#verifiedmessage").html() + ' ' + strLinkWhitelisted);
-                $("#verified").css('display', 'flex');
-            } else if (result.result.status === 'blocked') {
-                hideEverything();
-                blocked = true;
-                var strLinkBlocked = '';
-                if (result.result.type === 'domain' && 'category' in result.result.entries[0]) {
-                    $("#blacklistmessage").html('<b>' + encodeURI($("input").val().toLowerCase().replace('http://','').replace('https://','').replace('www.','').split(/[/?#]/)[0]) + '</b> was put on the blacklist for ' + result.result.entries[0].category.toLowerCase() + '.');
-                    strLinkBlocked = '<a id="details" href="/domain/' + encodeURI($("input").val()) + '">Details on this domain <i class="chevron right small icon"></i></a>';
-                } else if(result.result.type === 'address') {
-					          $("#blacklistmessage").html('<b>' + encodeURI($("input").val().toLowerCase()) + ' was put on the blacklist and is associated with '+ result.result.entries.length +' blocked domain(s).');
-					          strLinkBlocked = '<a id="details" href="/address/' + encodeURI($("input").val()) + '">Details on this address <i class="chevron right small icon"></i></a>';
-				        } else if(result.result.type === 'ip') {
-					          $("#blacklistmessage").html('<b>' + encodeURI($("input").val().toLowerCase().replace('http://','').replace('https://','').replace('www.','').split(/[/?#]/)[0]) + '</b> was put on the blacklist and is associated with '+ result.result.entries.length +' blocked domain(s)');
-					          strLink = '<a id="details" href="/ip/' + encodeURI($("input").val()) + '">Details on this domain <i class="chevron right small icon"></i></a>';
-				        }
-                $("#blacklistmessage").html($("#blacklistmessage").html() + ' ' + strLinkBlocked);
-                $("#blocked").css('display', 'flex');
-            }
-        });
-    });
+  $('.search-btn').click(function() {
+      $.getJSON("https://api.cryptoscamdb.org/v1/check/" + encodeURIComponent($("input").val().replace('http://','').replace('https://','').replace('www.','').split(/[/?#]/)[0]), function(result) {
```

Comment:
I think we should put `encodeURIComponent($("input").val().replace('http://','').replace('https://','').replace('www.','').split(/[/?#]/)[0])` to a variable so we don't replicate the stuff all thoughout this logic